src/status_im2/contexts/add_new_contact/views.cljs

```diff
@@ -42,7 +43,8 @@
                                    600)})]
         [quo/button
          (merge style/button-qr
-                {:on-press #(js/alert "TODO: to be implemented")})
+                {:on-press #(rf/dispatch [::qr-scanner/scan-code
```

Comment:
we shouldn't use namespaced keywords, and use status-im namespace in status-im2, there are two ways, simple, just add one more key to events , or replace namespaced keyword, but it think first one is simpler

Reply:
I think the namespaced keyword fix is not really the scope of this PR, given the namespace `status-im.qr-scanner.core` already exists and it's not following the guidelines at all in many other event names. I would say it would be better to do this refactoring separately.

Reply:
yes, so we could just add new event name , its not refactoring

Reply:
But @flexsurfer the event `::qr-scanner/scan-code` already exists and has a couple usages in the codebase. Creating a new one without changing the original would be unnecessary duplication just to get a different event keyword. if I understand correctly your suggestion I think that would be worse.